docs/controls/Gantt.md

```diff
@@ -80,6 +80,7 @@ ganttInstance.loadContent(/* Add "dataArray" JSON, shown below for example */);
 | showActionLegend | boolean  | true                                           | Toggle to show graph legend                                                          |
 | padding          | object   | `{ top: 10, bottom: 5, left: 100, right: 50 }` | Refer [Padding](Padding.md)                                                          |
 | clickPassThrough | object   | null                                           | Refer [Pass Through](#Pass-Through)                                                  |
+| pan              | object   | {}                                             | Set enabled to true inside pan to enable panning mode                                |
```

Comment:
Add a section for `Pan` property and explain what is has. 
Since this is a new property that is common for Graph, Gantt and Timeline. Lets create a new markdown for this similar to `Padding.md` and link it rather than duplicating it.

Reply:
ok, I will make a new Panning .md